lib/sky-pages-module-generator.js

```diff
@@ -29,7 +36,7 @@ function getSource(skyAppConfig) {
       if (/^http/.test(filePath)) {
         hasExternal = true;
       } else {
-        styles += `require('style-loader!${filePath}');\n`;
+        styles += `require('!style-loader!css-loader!${filePath}');\n`;
```

Comment:
The `css-loader` resolves all `@import url` within CSS files (the FontAwesome CDN URL wasn't being pulled in on this latest version of `@ngtools/webpack`).